packages/common/src/api/tan-query/useFollowing.ts

```diff
@@ -54,10 +54,12 @@ export const useFollowing = (
       })
       const users = userMetadataListFromSDK(data)
       primeUserData({ users, queryClient, dispatch })
-      return users
+      return users?.map((user) => user.user_id)
```

Comment:
Why is the `?` necessary? `userMetadataListFromSDK` should always be returning an array.
Also if it _is_ necessary, that would mean this function isn't guaranteed to return an array and that will change the behavior of the hook. So maybe consider default initializing it to an empty array.